src/proto/FIO.proto

```diff
@@ -0,0 +1,88 @@
+syntax = "proto3";
+
+package TW.FIO.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// A public blockchain address, such as {"BTC", "bc1qvy4074rggkdr2pzw5vpnn62eg0smzlxwp70d7v"}
+message PublicAddress {
+    string token_code = 1;
+    string address = 2;
+}
+
+// Different Actions
+message Action {
+    // Action for registering a FIO name; register_fio_address
+    message RegFioAddress {
+        // The FIO name to be registered. Ex.: "alice:trust"
+        string fio_address = 1;
+
+        // FIO address of the owner. Ex.: "FIO6m1fMdTpRkRBnedvYshXCxLFiC5suRU8KDfx8xxtXp2hntxpnf"
+        string owner_fio_public_key = 2;
+        
+        // Max fee to spend, can be obtained using get_fee API.
+        uint64 max_fee = 3;
+        
+        // The FIO name of the originating wallet, use DefaultTpid
+        string tpid = 4;
+
+        // Note: actor is not needed, computed from owner_fio_public_key
+    }
+
+    // Acion for adding public chain addresses to a FIO name; add_pub_address
+    message AddPubAddress {
+        // The FIO name already registered to the owner. Ex.: "alice:trust"
+        string fio_address = 1;
+
+        // List of public addresses to be registered, ex. {{"BTC", "bc1qv...7v"},{"BNB", "bnb1ts3...9s"}}
+        repeated PublicAddress public_addresses = 2;
+        
+        // Max fee to spend, can be obtained using get_fee API.
+        uint64 max_fee = 3;
```

Comment:
Just ```fee```?

Reply:
Max_fee is not too long, semantically correct, and in line with FIO API usage; keeping.

Reply:
It's confusing to me